Resources/Locale/en-US/_Goobstation/changeling/store/changeling-catalog.ftl

```diff
@@ -1,140 +1,125 @@
-    Costs 30 chemicals.
+    We expel impurities from our form, sobering us, and purging radiation and toxins.
+    Costs a small amount of chemicals.
 
 evolutionmenu-utility-eyesight-name = Augmented Eyesight
 evolutionmenu-utility-eyesight-desc =
-    Evolve additional features in your eyes, such as flash protection.
-    Cost-free.
+    We grow a tinted nictitating membrane over our eyes, protecting our vision from sudden bright lights.
+    Costs no chemicals.
 
 evolutionmenu-utility-biodegrade-name = Biodegrade
 evolutionmenu-utility-biodegrade-desc =
-    Vomit a caustic substance onto any restraints you may be wearing, allowing yourself to break free.
-    Using this ability while being grabbed will spit acid in your attackers face.
-    Costs 30 chemicals.
+    We spew a caustic substance, dissolving restraints and spraying those who hold onto us.
+    Costs a small amount of chemicals.
 
 evolutionmenu-utility-chameleon-name = Chameleon Skin
 evolutionmenu-utility-chameleon-desc =
-    Alter the pigment in your skin to match your surroundings, rendering you invisible.
-    Costs 20 chemicals.
+    Our skin pigmentation rapidly changes to suit our current environment.
+    Costs a small amount of chemicals.
 
 evolutionmenu-utility-stims-name = Ephedrine Overdose
 evolutionmenu-utility-stims-desc =
-    Inject a cocktail of stimulants into yourself, quickly removing any stuns and giving yourself a speed boost.
+    We inject a cocktail of stimulants into ourself, lessening the effects of incapacitation and allowing us to move at extremely fast speeds.
     Continuous injection is poisonous.
-    Costs 30 chemicals.
+    Costs a small amount of chemicals.
 
 evolutionmenu-utility-fleshmend-name = Fleshmend
 evolutionmenu-utility-fleshmend-desc =
-    Rapidly heal yourself of all bruises and burns.
-    Costs 35 chemicals.
+    Our flesh rapidly regenerates, healing our burns, bruises, and asphyxiation.
+    Costs a medium amount of chemicals.
 
 evolutionmenu-utility-lastresort-name = Last Resort
 evolutionmenu-utility-lastresort-desc =
-    Abandon your current body and escape in the form of a headslug.
-    Costs 20 chemicals.
+    We abandon our vessel, escaping in our vulnerable true self, to find a new body to occupy.
+    Costs a small amount of chemicals.
 
 evolutionmenu-utility-lesserform-name = Lesser Form
 evolutionmenu-utility-lesserform-desc =
-    Abandon your current form and turn into a sentient monkey.
-    Costs 20 chemicals.
+    We become lesser, transforming into a sentient monkey.
+    Costs a small amount of chemicals.
 
 evolutionmenu-utility-spacesuit-name = Space Adaptation
 evolutionmenu-utility-spacesuit-desc =
-    Get rid of useless tissue in order to facilitate space travel. A source of oxygen is still required for space walking.
-    Costs 20 chemicals.
+    We prepare our cells to resist barotrauma. We still require air to breathe.
```

Comment:
This ones petty because air works in this context but i dont personally like using air as a broad term for breathable. makes sense for humans and slimes but less so for vox and decapoids. might be better to say smth like "We still must breathe."

Reply:
adjusted to `We prepare our cells to resist barotrauma. We must still breathe.`